docs/05-technical-reference/00-custom-resources/apix-01-apirule.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: APIRule
+title: API Rule
 ---
 
 The `apirule.gateway.kyma-project.io` custom resource definition (CRD) is a detailed description of the kind of data and the format the API Gateway Controller listens for. To get the up-to-date CRD and show the output in the `yaml` format, run this command:
```

Comment:
(3) If the change mentioned in the previous comments is intentional, should it be applied here too?